### ISSUE TITLE
Open asset file does not work

### ISSUE THREAD

**Issue by Mzoltan32** [xenko3d/xenko#597]
**Release Type**: Official Release

**Version**: Version number 3.1

**Platform(s)**: Does the problem occur on Windows

**Describe the bug**
does not poen asset file because no assigned programs exist

**To Reproduce**
1. 

**Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem (use GitHub drag & drop)

**Log and callstacks**
If any, please attach here any log or callstack (preferably in a `.txt` file using GitHub drag and drop)

**Additional context**

**Comment by Kryptos-FR** [comment]
Please provide the steps to reproduce and remove any paragraph that don't add any useful information.

**Comment by Mzoltan32** [comment]
on a fresh installation open a map

then rightclick on an asset and select the open asset file option

**Comment by Kryptos-FR** [comment]
Have you read the documentation?

**Comment by Mzoltan32** [comment]
i was just reading tutorials, it seems there are no basal editor programs
bound to the asset file

**Comment by Eideren** [comment]
Cannot open an asset file which doesn't have a default program associated to it by the OS, we should probably have a modal show up instead of silently failing.

**Comment by mattiascibien** [comment]
Hello,

as a newcomer to this engine I wanted to try this issue but it seems that is already fixed as I get Windows 11 chooser to select which program to open.

Is it already fixed or could it happen on another OS version?

**Comment by manio143** [comment]
I don't think it has been fixed so it may be OS dependant or file extension dependent, or user setting dependent (e.g. selecting auto-behavior for the dialog which shows up the first time) - would be interesting to see if we can get any feedback from the OS if file open resulted in an application being started.

**Comment by mattiascibien** [comment]
Thanks for your answer. I will try to reproduce it during the day and try to see if I can provide a fix. Did not examine the code at the moment but I alway tought that a `System.Diagnostics.Process.Start(file)` would suffice.

**Comment by mattiascibien** [comment]
I have checked the code and found that the function to check is:
```csharp
public async Task OpenFile(string filePath, bool tryEdit)
        {
            try
            {
                filePath = filePath.Replace('/', '\\');
                if (!File.Exists(filePath))
                {
                    await ServiceProvider.Get<IDialogService>().MessageBox(Tr._p("Message", "You need to save the file before you can open it."), MessageBoxButton.OK, MessageBoxImage.Information);
                    return;
                }

                var process = new Process { StartInfo = new ProcessStartInfo(filePath) { UseShellExecute = true } };
                if (tryEdit)
                {
                    process.StartInfo.Verb = process.StartInfo.Verbs.FirstOrDefault(x => x.ToLowerInvariant() == "edit");
                }
                process.Start();
            }
            catch (Exception ex)
            {
                var message = $"{Tr._p("Message", "An error occurred while opening the file.")}{ex.FormatSummary(true)}";
                await ServiceProvider.Get<IDialogService>().MessageBox(message, MessageBoxButton.OK, MessageBoxImage.Error);
            }
        }
``` 

As far as I tried I do not see any problem in opening the file. @Mzoltan32 can you provide more info and a reproduction project to better check this?
